data/partials/home.yaml

```diff
@@ -23,94 +23,94 @@ nav_sections:
-      - title: RUM & Session Replay
-        link: real_user_monitoring/
-        icon: rum
-        desc: Visualize and analyze the performance of your front end applications
+        desc: Process, monitor, and archive your ingested logs
+      - title: Metrics
+        link: metrics/
+        icon: metric
+        desc: Explore, search, and create distributions for your metrics
+      - title: Mobile Application
+        link: mobile/
+        icon: mobile
+        desc: View Datadog alerts, incidents, and more on your mobile device
       - title: Network Monitoring
         link: network_monitoring/
         icon: network
-        desc: Gain visibility into your network traffic using tagged objects
+        desc: Use tagged objects to collect and graph data about your network traffic
+      - title: RUM & Session Replay
+        link: real_user_monitoring/
+        icon: rum
+        desc: Capture, observe, and analyze the user experience of your applications 
+      - title: Security Platform
+        link: security_platform/
+        icon: security-platform
+        desc: Detect threats and misconfigurations in your applications and infrastructure
       - title: Serverless
         link: serverless/
         icon: serverless
-        desc: End-to-end visibility into your serverless development lifecycle
-      - title: Mobile application
-        link: mobile/
-        icon: mobile
-        desc: View Datadog alerts and dashboards on your mobile device
-      - title: Datadog Watchdog
+        desc: Detect and resolve performance issues in your serverless applications
+      - title: Synthetic Monitoring
+        link: synthetics/
+        icon: synthetics
+        desc: Ensure uptime, flag regional issues, and test application performance
+      - title: Watchdog
         link: watchdog/
         icon: watchdog
-        desc: Automatically see trends in your applications & infrastructure metrics
-      - title: CI Visibility
-        link: continuous_integration/
-        icon: ci
-        desc: Track the performance and reliability of your tests and pipelines
-      - title: Database Monitoring
-        link: database_monitoring/
-        icon: dbm
-        desc: Gain deep visibility into database and query performance
+        desc: Observe trends and patterns in your APM and infrastructure metrics
   - nav_section:
     - name: 'Configuration'
     - navtiles:
-      - title: API
-        link: api/
-        icon: api
-        desc: Use the Datadog API
       - title: Account Management
         link: account_management/
         icon: cog
-        desc: Control your account settings and manage billing
+        desc: Access org-based settings and billing
+      - title: API
+        link: api/latest/
+        icon: api
+        desc: Try the Datadog API
+      - title: Developers
+        link: developers/
+        icon: dev-code
+        desc: Develop on the Datadog platform
```

Comment:
As far as reordering alphabetically goes, I don't think we should go this route. As Kari pointed out, it's only alphabetical in english, and we have a large number of users who view the site in other languages. Additionally, this is a big visual/structural change for people who rely on the existing order of the tiles, people who are referencing this page regularly are going to be thrown with everything being reordered.